claim/netdata-claim.sh.in

```diff
@@ -288,10 +301,25 @@ if [ "${HTTP_STATUS_CODE}" = "204" ] ; then
         if [ "$EUID" == "0" ]; then
             chown -R "${NETDATA_USER}:${NETDATA_USER}" ${CLAIMING_DIR} || (echo >&2 "Claiming failed"; set -e; exit 2)
         fi
+
+        # Rewrite the netdata.conf on the disk
+        T=$(mktemp)
+        sed -e 's/^[^#]*netdata cloud =/#\0/;s/^[^#]*cloud base url =/#\0/' @configdir_POST@/netdata.conf >$T
```

Comment:
You should check out the shellcheck warnings, I think it wants to turn `$T` to `"$T"` mostly.

Reply:
Yes. I will flip the setting names next and the default base url then start polishing.